test/apps/rails5.2/lib/shell.rb

```diff
@@ -94,4 +94,38 @@ def interpolated_in_percent_W
     # Should not warn
     system(*%W(foo bar #{value}))
   end
+
+  def completely_external
+    system(foo) # We assume this is safe and do not warn.
```

Comment:
@presidentbeef can you confirm that this is the behavior we want?

Reply:
Yes. This check is already pretty noisy. If we don't know what the input is, let's not report.